pom.xml

```diff
@@ -132,6 +131,16 @@
   <build>
     <pluginManagement>
       <plugins>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-compiler-plugin</artifactId>
+          <version>3.8.1</version>
+        </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-plugin-plugin</artifactId>
+          <version>3.6.0</version>
```

Comment:
Wouldn't it make more sense to update `<mavenVersion>3.0</mavenVersion>` to `3.6.3`? Other dependencies in the POM are tied to that property.

Reply:
We want to keep the plugin useful for as many maven version as possible. Most likely this plugin doesn't use any feature that requires 3.6.3. Recently we've decided to change the minimum for plugin from 3.0 to 3.1.0